nicegui/testing/user_interaction.py

```diff
@@ -75,3 +78,14 @@ def click(self) -> Self:
                     event_arguments = events.GenericEventArguments(sender=element, client=self.user.client, args=args)
                     events.handle_event(listener.handler, event_arguments)
         return self
+
+    def clear(self) -> Self:
+        """Clear the selected elements.
+
+        Note: All elements must have a ``value`` attribute)."""
+        assert self.user.client
+        with self.user.client:
+            for element in self.elements:
+                assert isinstance(element, (ValueElement))
+                element.value = ''
```

Comment:
```suggestion
                element.value = None
```
I wonder if `None` would be the more natural value for clearing a value element. That's the value assigned when using the "clearable" prop on Quasar inputs. And it's the initial default value for `ui.number` (where an empty string "" is not applicable). But the default for `ui.input` is "", so it's not 100% obvious what `clear()` is supposed to do. For the sake of user tests the difference between `None` and "" might be irrelevant though. For compatibility with `ui.number` I'd vote for `None`.